runagent/utils/agent_id.py

```
@@ -0,0 +1,185 @@
+        try:
+            with open(folder_path / "requirements.txt", "r") as f:
+                content = f.read().lower()
+                if "langchain" in content:
+                    return "langchain"
+                elif "crewai" in content:
+                    return "crewai"
+                elif "autogen" in content:
+                    return "autogen"
+                elif "openai" in content:
+                    return "openai"
+        except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)